docs/index.zh.md

```diff
@@ -24,38 +24,35 @@ under the License.
+- **编程指南**: 你可以从 [基本 API 概念](dev/api_concepts.html), [DataStream API](dev/datastream_api.html) 以及 [DataSet API](dev/batch/index.html) 着手学习如何编写你的第一个 Flink 作业。
 
-- **Tutorials**: 
-  * [Implement and run a DataStream application](./tutorials/datastream_api.html)
-  * [Setup a local Flink cluster](./tutorials/local_setup.html)
+## 部署
+在线上环境运行你的 Flink 作业之前，请阅读 [生产环境注释事项检查清单](ops/production_ready.html)。
 
-- **Programming Guides**: You can read our guides about [basic API concepts](dev/api_concepts.html) and the [DataStream API](dev/datastream_api.html) or the [DataSet API](dev/batch/index.html) to learn how to write your first Flink programs.
+## 发布日志
 
-## Deployment
+发布日志包含了 Flink 版本之间的重大更新。请在你升级 Flink 之前仔细阅读相应的发布日志。
 
-Before putting your Flink job into production, read the [Production Readiness Checklist](ops/production_ready.html).
+* [Flink 1.8 的发布日志](release-notes/flink-1.8.html)。
+* [Flink 1.7 的发布日志](release-notes/flink-1.7.html)。
+* [Flink 1.6 的发布日志](release-notes/flink-1.6.html)。
+* [Flink 1.5 的发布日志](release-notes/flink-1.5.html)。
 
-## Release Notes
+## 外部资源
 
-Release notes cover important changes between Flink versions. Please carefully read these notes if you plan to upgrade your Flink setup to a later version. 
+- **Flink Forward**: 历史大会的所有演讲均可在 [Flink Forward](http://flink-forward.org/) 官网以及 [YouTube](https://www.youtube.com/channel/UCY8_lgiZLZErZPF47a2hXMA)找到。[使用 Apache Flink 进行高可靠的流处理](http://2016.flink-forward.org/kb_sessions/robust-stream-processing-with-apache-flink/) 可以作为你第一个学习的资源。
 
-* [Release notes for Flink 1.8](release-notes/flink-1.8.html).
-* [Release notes for Flink 1.7](release-notes/flink-1.7.html).
-* [Release notes for Flink 1.6](release-notes/flink-1.6.html).
-* [Release notes for Flink 1.5](release-notes/flink-1.5.html).
+- **培训**: [培训资料](https://training.ververica.com//) 包含讲义，练习以及示例程序。
 
-## External Resources
-
-- **Flink Forward**: Talks from past conferences are available at the [Flink Forward](http://flink-forward.org/) website and on [YouTube](https://www.youtube.com/channel/UCY8_lgiZLZErZPF47a2hXMA). [Robust Stream Processing with Apache Flink](http://2016.flink-forward.org/kb_sessions/robust-stream-processing-with-apache-flink/) is a good place to start.
-
-- **Training**: The [training materials](http://training.data-artisans.com/) from data Artisans include slides, exercises, and sample solutions.
-
-- **Blogs**: The [Apache Flink](https://flink.apache.org/blog/) and [data Artisans](https://data-artisans.com/blog/) blogs publish frequent, in-depth technical articles about Flink.
+- **博客**: [Apache Flink](https://flink.apache.org/blog/) 以及 [ververica](https://www.ververica.com/blog) 的博客会经常更新一些有关 Flink 的技术文章。
```

Comment:
ververica -> Ververica

Could you also update corresponding English file for this?